geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/StatsRunner.scala

```diff
@@ -9,10 +9,11 @@
 package org.locationtech.geomesa.accumulo.data.stats
 
 import java.io.Closeable
+import java.time.temporal.ChronoUnit
+import java.time.{Clock, Instant}
```

Comment:
Yes, I think we should slowly move away from joda entirely now that java 8 has better time libs. I also ran into joda conflicts with the distributed runtime jar.